BackEndFlask/models/role.py

```diff
@@ -42,9 +42,9 @@ def load_existing_roles():
     create_role("Researcher")     # 1
     create_role("SuperAdmin")     # 2
     create_role("Admin")          # 3
-    create_role("TA/Instructor")  # 4
-    create_role("Student")        # 5
-    create_role("Teams")          # 6
+    create_role("Instructor")     # 4
```

Comment:
Why did you separate TA/Instructor?

Reply:
I agree the Instructor and TA have different relationships to a course. However, separating the single role into two roles shifts the ids. Thus resulting in breaking the rest of the project because the project assumes the original role ids. The reason we have TA/Instructor was per request from the client of needing a role that says exactly those words. The role itself is really just a TA. The Admin role is really the Instructor. So if we were to make a change we would just remove Instructor from the name.

Reply:
Okay, I switched it back.